vocabularies/trs.ttl

```diff
@@ -0,0 +1,178 @@
+
+The Julian Day Number (JDN) is the integer assigned to a whole solar day in the Julian day count starting from noon Universal time, with Julian day number 0 assigned to the day starting at noon on Monday, January 1, 4713 BC, proleptic Julian calendar (November 24, 4714 BC, in the proleptic Gregorian calendar), a date at which three multi-year cycles started (which are: Indiction, Solar, and Lunar cycles) and which preceded any dates in recorded history. For example, the Julian day number for the day starting at 12:00 UT on January 1, 2000, was 2 451 545. """ ;
+  skos:prefLabel "Julian day" ;
+  skos:topConceptOf trs:conceptScheme ;
+.
+trs:MayaCalendar
+  a skos:Concept ;
+  a time:TRS ;
+  rdfs:seeAlso <https://en.wikipedia.org/wiki/Maya_calendar> ;
+  skos:definition "The Maya calendar is a system of calendars used in pre-Columbian Mesoamerica and in many modern communities in the Guatemalan highlands, Veracruz, Oaxaca and Chiapas, Mexico." ;
+  skos:prefLabel "Maya Calendar" ;
+  skos:topConceptOf trs:conceptScheme ;
+.
+trs:ParliamentsOfAustraliaTimeline
+  a skos:Concept ;
+  a time:TRS ;
+  dct:source <https://en.wikipedia.org/wiki/Chronology_of_Australian_federal_parliaments> ;
+  rdfs:seeAlso <https://en.wikipedia.org/wiki/Parliament_of_Australia> ;
+  skos:definition """The Government of Australia is the government of the Commonwealth of Australia, a federal parliamentary constitutional monarchy. It is also commonly referred to as the Australian Government, the Commonwealth Government, Her Majesty's Government, or the Federal Government.
+
+The Parliament of Australia (officially the Federal Parliament, also called the Commonwealth Parliament) is the legislative branch of the government of Australia. It consists of three elements: the Crown (represented by the Governor-General), the Senate and the House of Representatives. The combination of two elected chambers, in which the members of the Senate represent the states and territories while the members of the House represent electoral divisions according to population, is modelled on the United States Congress. Through both chambers, however, there is a fused executive, drawn from the Westminster system.
+
+This timeline of Parliaments of Australia starts in 1901 (Gregorian Caledar) when Australia was formed as an country from pre-existing colonies that where Dominions of the British Empire and were not sovereign states themselves. The 1901 Australian federal election for the inaugural Parliament of Australia was held in Australia on Friday 29 March and Saturday 30 March 1901 after which Edmond Barton was sworn in as the inaugural Prime Minister.
+
+Epoch within this timeline are numbered 1st, 2nd etc. up to (as of October, 2019, Gegorian Calendar) the 44th.""" ;
```

Comment:
Is this still part of the definition? Should be on a single line.
Also ends with many quotation marks.

Reply:
as above